src/standalone/InlineSwitch/index.tsx

```diff
@@ -0,0 +1,120 @@
+import React, { useCallback } from "react";
+import { Grid, Typography, withStyles, Theme, Switch } from "@material-ui/core";
+import { makeStyles } from "@material-ui/core/styles";
+
+export interface InlineSwitch {
+	/**
+	 * Value for switch position
+	 */
+	switchValue: boolean;
+	/**
+	 * Set value for switch position
+	 */
+	setSwitchValue: (checked: boolean) => void;
```

Comment:
rename to `onChange: (checked: boolean) => void`. also lacks docs for param checked

Reply:
@TQjona , Changes updated. Please Review.